### ISSUE TITLE
Using Redis storage without user/password

### ISSUE THREAD

**Issue by reeslo** [itsgoingd/clockwork#711]
There is an issue when using Redis storage but without authentication

![image](https://github.com/user-attachments/assets/7c4b1966-0201-42e6-9c3b-9b62a192c342)

```
A PHP Error was encountered

Severity: Warning

Message: Redis::auth(): When passing an array as auth it must have one or two elements!

Filename: Storage/RedisStorage.php

Line Number: 55

Backtrace:
```

Also if the database is not set it must be `0` like
```php
		$redis->select($connection['database'] ?? 0);
```

```
A PHP Error was encountered

Severity: Warning

Message: Undefined array key "database"

Filename: Storage/RedisStorage.php

Line Number: 58
```

**Comment by itsgoingd** [comment]
Thanks for the report, I'll try to fix these soon.

**Comment by reeslo** [comment]
You're welcome! I've fixed it by adding fake informations :)

**Comment by reeslo** [comment]
Hi, the ui interface seems not usable when Redis driver is set.

**Comment by itsgoingd** [comment]
Oops, forgot about this one, this is now fixed in master and will be released in Clockwork 5.3 (https://github.com/itsgoingd/clockwork/commit/a46f56a9d0e6e498dfd0f44214299149f235b103).
